tests/unit/moduleapi/keyspecs.tcl

```diff
@@ -8,12 +8,12 @@ start_server {tags {"modules"}} {
         # Verify (first, last, step) and not movablekeys
         assert_equal [lindex $reply 2] {module}
         assert_equal [lindex $reply 3] 1
-        assert_equal [lindex $reply 4] 3
-        assert_equal [lindex $reply 5] 2
+        assert_equal [lindex $reply 4] 2
+        assert_equal [lindex $reply 5] 1
         # Verify key-spec auto-generated from the legacy triple
         set keyspecs [lindex $reply 8]
         assert_equal [llength $keyspecs] 1
-        assert_equal [lindex $keyspecs 0] {flags {} begin_search {type index spec {index 1}} find_keys {type range spec {lastkey 2 keystep 2 limit 0}}}
+        assert_equal [lindex $keyspecs 0] {flags {RW access update} begin_search {type index spec {index 1}} find_keys {type range spec {lastkey 1 keystep 1 limit 0}}}
```

Comment:
I added this unusual (1,3,2) to test that the algorithm creating a keyspec from the triple can handle step other than 1. I'm not sure it's tested elsewhere.

Reply:
Ok, I'll revert, add a comment, and arity check. 
The reason I changed it was because I also had to execute that command with sufficient arguments (no just introspect the specs)